src/components/topic/Description.js

```diff
@@ -56,21 +46,24 @@ const TopicDescription = styled.div`
     margin: auto;
     border-top: 2px solid ${colors.red.rustyRed};
   }
+
+  /* Since the component corresponding to unstyled type uses a className to address text-align as justify. 
+   * Boost the specificity of the override style here */
```

Comment:
May we change commenting style to
```
/**
 *  Since the component corresponding to unstyled type uses a className to address text-align as justify. 
 *  Boost the specificity of the override style here
 */
```
or 
```
// Since the component corresponding to unstyled type uses a className to address text-align as justify. 
// Boost the specificity of the override style here
```

Reply:
Sure.